sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -27,12 +28,12 @@ export interface AnalyzedTokenInfo {
 
 // @public
 export interface AnalyzeRequest {
-    analyzerName?: string;
-    charFilters?: string[];
+    analyzerName?: LexicalAnalyzerName;
```

Comment:
Ignore me if these are all aliases to string, but if they are unions or some other narrower type this would be a breaking change?

I won't comment on every change here yet but we should not have breaking changes in a minor release so please do validate that 👍

Reply:
Some of these changes are just named string aliases, some of them are string literal unions where they're output-only parameters. But only purportedly. I mentioned this elsewhere, but it's probably not worth the maintenance burden to manually track down which extensible enums are output-only, and I'd have to think about whether or not it's worth making a Swagger parser to automate this.